package.json

```diff
@@ -14,9 +14,9 @@
     "device detection",
     "privacy"
   ],
-  "main": "dist/botd.cjs.js",
-  "module": "dist/botd.esm.js",
-  "types": "dist/botd.d.ts",
+  "main": "dist/main.esm.js",
+  "module": "dist/main.esm.js",
+  "types": "dist/index.d.ts",
```

Comment:
These lines are incorrect. The files in `dist` have other names. The "main" file must be in the CommonJS format.